### DISCUSSION TITLE
Support times with GitHub Education

### DISCUSSION THREAD

**Discussion by guldkage** [community/community#52635] [GitHub Education] [answered]
Anyone knows how long it takes for GitHub Education team to answer my ticket? It's been over 20 days with no answer..

**Comment by samus-aran** [comment, marked as answer]
Hi @guldkage 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐ 
We are seeing a higher volume of student developer pack inquiries coming through. We are working our way through addressing these as fast as possible.
I can share that your request has been passed onto the GitHub Education team and will be processed as soon as possible, but I am afraid I do not have a timeline to share for when you can expect this to happen.

**Reply by guldkage** [reply]
Thank you. I hope to get an answer soon.
